web/src/components/tasks/business-formation/business/NonprofitProvisions.tsx

```diff
@@ -0,0 +1,169 @@
+                control={
+                  <Radio
+                    required={true}
+                    data-testid={`${fieldName}-${value}`}
+                    color={hasError ? "error" : "primary"}
+                  />
+                }
+                label={getRadioLabel(value)}
+              />
+            ))}
+          </RadioGroup>
+          <FormHelperText>{hasError ? Config.formation.general.genericErrorText : ""}</FormHelperText>
+        </FormControl>
+      </WithErrorBar>
+    );
+  };
+
+  return (
+    <>
+      <div className="flex flex-column mobile-lg:flex-row mobile-lg:flex-align-center margin-bottom-2">
+        <div role="heading" aria-level={2} className="h3-styling margin-bottom-0">
+          {Config.formation.nonprofitProvisions.label}
+        </div>
+      </div>
+
+      {getRadio({
+        fieldName: "hasNonprofitBoardMembers",
+        title: Config.formation.fields.hasNonprofitBoardMembers.body,
+        values: ["true", "false"],
+      })}
+
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        getRadio({
+          fieldName: "nonprofitBoardMemberQualificationsSpecified",
+          title: Config.formation.fields.nonprofitBoardMemberQualificationsSpecified.body,
+          values: ["IN_BYLAWS", "IN_FORM"],
+        })}
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        state.formationFormData.nonprofitBoardMemberQualificationsSpecified === "IN_FORM" &&
+        getTextField("nonprofitBoardMemberQualificationsTerms")}
+
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        getRadio({
+          fieldName: "nonprofitBoardMemberRightsSpecified",
+          title: Config.formation.fields.nonprofitBoardMemberRightsSpecified.body,
+          values: ["IN_BYLAWS", "IN_FORM"],
+        })}
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        state.formationFormData.nonprofitBoardMemberRightsSpecified === "IN_FORM" &&
+        getTextField("nonprofitBoardMemberRightsTerms")}
+
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        getRadio({
+          fieldName: "nonprofitTrusteesMethodSpecified",
+          title: Config.formation.fields.nonprofitTrusteesMethodSpecified.body,
+          values: ["IN_BYLAWS", "IN_FORM"],
+        })}
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        state.formationFormData.nonprofitTrusteesMethodSpecified === "IN_FORM" &&
+        getTextField("nonprofitTrusteesMethodTerms")}
+
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        getRadio({
+          fieldName: "nonprofitAssetDistributionSpecified",
+          title: Config.formation.fields.nonprofitAssetDistributionSpecified.body,
+          values: ["IN_BYLAWS", "IN_FORM"],
+        })}
+      {state.formationFormData.hasNonprofitBoardMembers &&
+        state.formationFormData.nonprofitAssetDistributionSpecified === "IN_FORM" &&
+        getTextField("nonprofitAssetDistributionTerms")}
+    </>
```

Comment:
[sand] A lot of duplicated logical comparisons here, like `{state.formationFormData.hasNonprofitBoardMembers &&` for one, and 

```
{state.formationFormData.hasNonprofitBoardMembers &&
        state.formationFormData.X === "IN_FORM" &&
```

Solid refactoring opportunity to make this less repetitive and more readable.

Reply:
Done